README.md

```diff
@@ -67,38 +74,65 @@ The [track anatomy documentation](https://github.com/exercism/docs/blob/master/l
 ```bash
 ├── .gitignore
 ├── .github
-│ └── workflows
-│     └── tests.yml
+│   └── workflows
+│       └── tests.yml
 ├── LICENSE
 ├── README.md
 ├── bin
-│ └── fetch‐configlet
+│   └── fetch‐configlet
+├── concepts
+│   └── basics
+│   │   ├── about.md
+│   │   ├── introduction.md
+│   │   └── links.json
+│   └── ...
 ├── config.json
 ├── docs
-│ ├── ABOUT.md
-  ├── EXERCISE_README_INSERT.md
-│ ├── INSTALLATION.md
-│ ├── LEARNING.md
-│ ├── RESOURCES.md
-│ └── TESTS.md
+│   ├── ABOUT.md
+│   ├── EXERCISE_README_INSERT.md
+│   ├── INSTALLATION.md
+│   ├── LEARNING.md
+│   ├── RESOURCES.md
+│   └── TESTS.md
 └── exercises
-  └── accumulate
-  │ ├── package.yaml
-  │ ├── stack.yaml
-  │ ├── src
-  │ │ └── Accumuĺate.hs
-  │ ├── test
-  │ │ └── Tests.hs
-  │ └── .meta
-  │   ├── examples
-  │   │ └── success-standard
+  ├── concept
```

Comment:
vertical line extending below `exercises` still is not aligned with the first `e` of `exercises`